Proiect-VS/assets/css/new-evaluation.less

```diff
@@ -0,0 +1,162 @@
+body {
+    position: relative;
+    margin: auto;				
+}
+
+.skin_body {
+    background-color: #f7f7f7;
+    font-family: Arial;
+}
+
+header {
+    height: 70px;
+    border-bottom: 1px solid grey;
+    font-size: 25px;
+}
+        
+footer {
+    height: 70px; 
+    line-height: 70px;
+    vertical-align: middle;
+    text-align: right;
+    padding-right: 50px;			
+    margin-top: 10px;
+    position: absolute;
+    left: 0;
+    right: 0;
+    float: left;
+}	
+
+.skin_footer {
+    background-color: #bcbcaf;
+    color: #686865;	
+}
+
+.logo {
+    width: auto;
+    height: 50px;
+    line-height: 70px;
+    vertical-align: middle;
+}
+
+.menu_button {
+    vertical-align: middle;
+    line-height: 70px;
+    margin-left: 100px;
+    margin-right: 15px;
+}
+
+.current_menu_button {
+    border-bottom: 9px solid #21bcff; 
+    padding-bottom: 12px;
+}
+
+a:link, a:visited {
+    text-decoration: none;
+    color: black;
+}
+
+.evaluation_form {
+    padding-top: 55px;
+    margin: 0 auto ;
+    width: 1000px;
+    height: 40px;
+}
+
+.evaluation_form_header {
+    border: none;
+    border-bottom: 3px solid #f7f7f7;
+    height: 30px;
+    width: 300px;
+    margin: 7px;
+}
+
+.evaluation_form_table {
+    border-collapse: collapse;
+    width: 1000px;
+    margin: 20.750px 0px;
+    text-align: center;
+}
+
+.evaluation_form_table_header {
+    height: 40px;
+    width: 935px;
+}
+
+.skin_evaluation_form_table_header {
+    background-color: #21bcff;
+    color: white;
+    font-size: 14px;
+}
+
+.evaluation_form_table_content {
+    border: 1px solid black;
+    height: 40px;
+    width: 934px;
+}
+
+.evaluation_form_submit {
+    float: right;
+    width: 150px;
+    height: 50px;
+    border: none;
+    border-radius: 2px;
+    box-shadow: 0px 3px 4px grey;
+    margin-bottom: 10px;
+    margin-top: 10px;
+}
+
+@media only screen and (max-device-width: 480px) {
+      .evaluation_form_submit {text-transform: uppercase; }
+}
+
+.skin_evaluation_form_submit {
+    background-color: #21bcff;
```

Comment:
good job at doing this separation. You should also check if some of these props combinations aren't used elsewhere in which case you can further abstract them